Source/JavaScriptCore/Scripts/copy-profiling-data.sh

```diff
@@ -1,14 +1,19 @@
-#!/bin/sh
+#!/bin/sh -e
 
 # Decompresses and copies PGO profiles from WebKitAdditions to a derived folder.
 
 if [ "${CLANG_USE_OPTIMIZATION_PROFILE}" = YES ]; then
-    compression_tool -v -decode -i "${SCRIPT_INPUT_FILE_0}" -o "${SCRIPT_OUTPUT_FILE_0}" && exit
-    if [ "${CONFIGURATION}" = Production ]; then
-        echo "error: ${SCRIPT_INPUT_FILE_0} failed to extract"
-        exit 1
+    eval $(stat -s "${SCRIPT_INPUT_FILE_0}")
+    if [ ${st_size} -lt 1024 ]; then
+        message="${SCRIPT_INPUT_FILE_0} is <1KB, is it a Git LFS stub? Ensure this file was checked out on a machine with git-lfs installed."
```

Comment:
I'm not sure if we want to command with "ensure" in a non-production build. Maybe describing the consequences would help people decide if they care.

Does this script get executed for open source builds, where there isn't even any action to take? We probably don't want to warn there.

Reply:
Good point!

> Does this script get executed for open source builds, where there isn't even any action to take? We probably don't want to warn there.

`$(CLANG_USE_OPTIMIZATION_PROFILE)` is only enabled on internal SDKs, so the script is a no-op in open source.